web/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "short_name": "wallabag",
+  "name": "Save and classify articles. Read them later. Freely.",
```

Comment:
Ideally this would be translated.

Reply:
Yeah but it'll be hard to translate it inside the json.